src/components/SignInForm.test.js

```diff
@@ -1,26 +1,18 @@
 import React from "react";
 import { mount } from "enzyme";
-import { SignInFormUnconnected } from "./SignInForm";
+import SignInFormUnconnected from "./SignInForm";
 
 describe("SignInForm", () => {
-  it.skip("signs in users", done => {
-    const mockFirebase = {
-      login: jest.fn(() => Promise.resolve({ email: "test@example.com" }))
-    };
-    const component = mount(
-      <SignInFormUnconnected
-        firebase={mockFirebase}
-        onSuccess={user => {
-          expect(mockFirebase.login).toBeCalledWith({
-            email: "test@example.com",
-            password: "password"
-          });
-          expect(user.email).toEqual("test@example.com");
-          done();
-        }}
-      />
+  let component;
+  const userSignIn = jest.fn(({ email, password }) => {
+    Promise.resolve();
+  });
+  beforeEach(() => {
+    component = mount(
+      <SignInFormUnconnected firebase={jest.fn()} userSignIn={userSignIn} />
```

Comment:
`SignInFormUnconnected` no longer takes a `firebase` property at all now, so this can be removed. (At some point it was refactored into a component and container, so the component is now purely UI.)

Reply:
I've removed the firebase prop, tests still pass.